composer/spec/dependabot/composer/update_checker_spec.rb

```diff
@@ -427,39 +421,6 @@
       it { is_expected.to be_nil }
     end
 
-    context "with a PEAR dependency (composer v1)" do
```

Comment:
`composer` `2` doesn't support PEAR, so we're implicitly dropping support for it when we drop `composer 1`. However, PEAR usage has been mostly supplanted by Packagist. 

This is confirmed by our metrics which show low usage of `composer 1`. And of those, I'm sure only a fraction will be using PEAR, so this should have hardly any user impact.